todo/views.py

```diff
@@ -0,0 +1,58 @@
+from django.http import HttpResponseRedirect
+from django.urls import reverse_lazy
+from django.views import generic
+
+from todo.models import Task, Tag
+from todo.forms import TaskForm, TagForm
+
+
+class TaskListView(generic.ListView):
+    model = Task
+
+
+class TaskCreateView(generic.CreateView):
+    model = Task
+    form_class = TaskForm
+    success_url = reverse_lazy("todo:task-list")
+
+
+class TaskUpdateView(generic.UpdateView):
+    model = Task
+    form_class = TaskForm
+    success_url = reverse_lazy("todo:task-list")
+
+
+class TaskDeleteView(generic.DeleteView):
+    model = Task
+    success_url = reverse_lazy("todo:task-list")
+
+
+def completed_status_change(request, pk):
+    task = Task.objects.get(id=pk)
+    if task.completed is True:
+        task.completed = False
+    else:
+        task.completed = True
```

Comment:
Would not it will be better?
```suggestion
    task.completed = not task.completed
```